account_invoice_production_lot/README.rst

```diff
@@ -1,65 +1,96 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl
-   :alt: License: AGPL-3
-
 =======================
 Invoice Production Lots
 =======================
 
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoice--reporting-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoice-reporting/tree/10.0/account_invoice_production_lot
+    :alt: OCA/account-invoice-reporting
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoice-reporting-10-0/account-invoice-reporting-10-0-account_invoice_production_lot
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/94/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
 This module shows, for each (customer) invoice line, the delivered production
 lots.
+Moreover, on the invoice report, these latter will be displayed with their corresponding delivered quantities.
+
+**Table of contents**
+
+.. contents::
+   :local:
 
 Configuration
 =============
 
-* Go to **Inventory > Configuration > Settings**, and activate option "Track
-  lots or serial numbers" in order to manage lots in your instance.
-* Go to **Sales > Sales > Products** and select or create a product and check
-  that product has enabled its tracking by lots in **Inventory** tab.
+* Go to **Inventory > Configuration > Settings**, and activate option *Track
+  lots or serial numbers* in order to manage lots in your instance
+* Go to **Sales > Sales > Products** and select or create a product stockable and check
+  that product has enabled its tracking by lots in **Inventory** tab and set **Invoicing Policy** as **Delivered quantities**
```

Comment:
Please rephrase:

Go to **Sales > Sales > Products** and select or create a stockable product: check
  that the product has **Tracking** set to **By lots** (in the **Inventory** tab) and **Invoicing Policy** set to **Delivered quantities** (in the **Invoicing** tab)

Reply:
tracking by serial number is supported too

Reply:
Done in https://github.com/OCA/account-invoice-reporting/pull/85/commits/f6248e57d3c2a183d6db7fb19dc48145f4421414

Reply:
You have to change https://github.com/OCA/account-invoice-reporting/pull/85/files#diff-69ac3ca1b6968d716b421a1f0537d6d8R68 in order to support Serial number, otherwise the quantity is not shown

Reply:
When *By Unique Serial Number* is set, the quantity will be always equal to 1. It seems to me useless show it in that case.

Reply:
Ok, showing the quantity for S/N is actually useless, anyway you could add a note about them in https://github.com/OCA/account-invoice-reporting/pull/85/files#diff-69ac3ca1b6968d716b421a1f0537d6d8R74

Reply:
Done in https://github.com/OCA/account-invoice-reporting/pull/85/commits/526c171623466dc80b7d401b3c287352f937b897